wsat-simple/src/test/java/org/jboss/as/quickstarts/wsat/simple/ClientStub.java

```diff
@@ -35,6 +37,6 @@
  */
 @Qualifier
 @Retention(RUNTIME)
-@Target({ TYPE, FIELD })
+@Target({TYPE, METHOD, FIELD, PARAMETER})
```

Comment:
You mentioned this was something you were not sure about. i think it's right:

TYPE: allows the injected class type to be annotated. 
METHOD: Used with @Produces on factory methods
FIELD: Typical use case of injecting into a class field
PARAMETER: One use would be to populate fields on a producer method.